docs/components/AppLayout.js

```diff
@@ -15,15 +17,47 @@ type Props = {|
 |};
 
 export default function AppLayout({ children, colorScheme, isHomePage }: Props): Node {
+  const { isMobile } = useDocsDeviceType();
+  const { isSidebarOpen } = useNavigationContext();
+
   const footerColor =
     colorScheme === 'dark' ? 'var(--color-gray-roboflow-700)' : 'var(--color-orange-firetini-0)';
-  return (
+
+  return isMobile && isSidebarOpen ? (
+    <Box
+      position="absolute"
+      top
+      bottom
+      left
+      right
+      overflow="scroll"
+      display="block"
+      mdDisplay="none"
+    >
+      <DocsSideNavigation />
+    </Box>
+  ) : (
     <Box minHeight="100vh" color="default">
       <Header />
-
       <Box mdDisplay="flex">
         <Box minWidth={MIN_NAV_WIDTH_PX}>
-          <Navigation />
+          {isSidebarOpen ? (
```

Comment:
@ayeshakmaz this is a change you can adopt in your PR to prevent further conflicts. 

Basically I moved the isSidebarOpen ternary out of Navigation

I also renamed Navigation to DocsSideNavigation.